scripts/zones/Ilrusi_Atoll/mobs/Carrion_Leech.lua

```diff
@@ -3,21 +3,16 @@
 --  Mob: Carrion Leech
 -----------------------------------
 local ID = require("scripts/zones/Ilrusi_Atoll/IDs")
+local zoneUtil = require("scripts/zones/Ilrusi_Atoll/globals/zoneUtil")
```

Comment:
Just a thought after seeing these.  Do we want to keep a globals underneath the zone, or perhaps move files like these out in a similar manner to job_utils?